app/presentation/RoomItem.js

```diff
@@ -203,6 +203,11 @@ export default class RoomItem extends React.Component {
 			prefix = `${ lastMessage.u.username }: `;
 		}
 
+		if (lastMessage.msg && lastMessage.attachments && Object.keys(lastMessage.attachments).length > 0) {
+			lastMessage.msg = lastMessage.msg.replace(/^\[ ]\(.+?\)/, '');
+			prefix = prefix.replace(/:/, ` (${ I18n.t('Replied') }):`);
```

Comment:
Remove it. It's cleaner without it.

Reply:
Also remove `You:` from prefix if `room.t === 'd'`

Reply:
done